src/intro/intro_readme.md

```diff
@@ -0,0 +1,22 @@
+![header](./img/intro_header.png)
+
+# Welcome to the ThreeFold Grid Manual!
+
+Welcome to the __ThreeFold Grid Manual__, your comprehensive guide to understanding and harnessing the power of the ThreeFold Grid. This manual has been meticulously created to provide you with the knowledge and tools you need to navigate and make the most of this revolutionary decentralized grid infrastructure.
+
+## About the ThreeFold Grid
+
+The ThreeFold Grid is a global, peer-to-peer network that aims to create a more sustainable and equitable digital world. It is built on the principles of decentralization, empowering individuals and organizations to take control of their digital assets, data, and interactions. The ThreeFold Grid is designed to be highly scalable, secure, and environmentally friendly, offering a robust and efficient alternative to traditional centralized cloud and computing services.
+
+Within this manual, you will find a wealth of information on how to leverage the capabilities of the ThreeFold Grid. Whether you are an individual user, a farmer, a developer, or a business entity, this manual will guide you through the process of accessing, deploying, and managing resources on the grid.
+
+## Key Sections
+
+- [The Grid Concepts](./concepts/concepts_readme.md): This section provides a brief introduction to the ThreeFold Grid, its core concepts, and its key benefits. It will help you understand the fundamental principles behind the grid and set the stage for further exploration.
```

Comment:
Incorrect path

![image](https://github.com/threefoldtech/info_grid/assets/101194226/8fbd6766-d48b-4c20-8237-e6e7eac82660)

i think it should be `../concepts/concepts_readme.md`